src/rtextures.c

```diff
@@ -2073,6 +2073,65 @@ void ImageColorReplace(Image *image, Color color, Color replace)
 
     ImageFormat(image, format);
 }
+// Flood-fills from the given position
+void ImageFloodFill(Image *dst, int startPosX, int startPosY, Color color){
```

Comment:
Please, could you review [raylib coding conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md)?.